content/_plugins.md

```diff
@@ -412,3 +412,189 @@ Valid values are `tenant` or `global`.
 
 ### Response
 A `Plugin` resource.
+
+## The Plugins Update Resource
+
+### Attributes:
+
+Attribute | Type | Description
+--------- | ------- | -------
+`id` | string | A unique identifier for the plugins update.
+`state` | string | The state of this update (successful, failed, updating, etc...).
+`blueprint_id` | string | The id of the blueprint to its deployments.
```

Comment:
The id of the blueprint which all of it's deployments will be updated.
Or something like that